src/components/App/App.js

```diff
@@ -30,16 +38,14 @@ class App extends Component {
         <nav>
           <h1>Rancid Tomatillos</h1>
         </nav>
-        {!this.state.movies.length && <h3>Loading...</h3>}
+        {this.state.error && <h3>{this.state.error}</h3>}
```

Comment:
When error message pops up for a single movie, all movies still display.  Let's look into styling this differently maybe?